lib/hooks/useInsightMembers.ts

```diff
@@ -0,0 +1,81 @@
+import useSWR, { Fetcher } from "swr";
+import useSupabaseAuth from "./useSupabaseAuth";
+import publicApiFetcher from "lib/utils/public-api-fetcher";
+interface PaginatedInsightMembers {
+  data: DbInsightMember[];
+  meta: Meta;
+}
+
+const useInsightMembers = (page_id: number) => {
```

Comment:
Let's try to keep the naming/casing for variables in the codebase consistent

```suggestion
const useInsightMembers = (insightId: number) => {
```